gammapy/image/maps.py

```diff
@@ -341,12 +341,13 @@ def reproject(self, refheader, mode='interp', *args, **kwargs):
         skymap : `SkyMap`
             Skymap reprojected onto ``refheader``.
         """
+
         from reproject import reproject_interp
         out = reproject_interp(
             input_data=self.to_image_hdu(),
             output_projection=refheader,
             *args,
-            **kwargs,
+            **kwargs
```

Comment:
@JouvinLea - This line was fixed in `master` already. Can you please rebase this PR to resolve the merge conflict?